statesman-engine/src/test/java/io/appform/statesman/engine/handlebars/HandleBarsServiceTest.java

```diff
@@ -271,6 +271,19 @@ public void testAdd() {
         Assert.assertEquals("7", hb.transform("{{add days 5}}", node));
     }
 
+    @Test
+    @SneakyThrows
+    public void testSumMultipleNumbers() {
+        val hb = new HandleBarsService();
+        val node = Jackson.newObjectMapper()
+                .createObjectNode()
+                .put("Q1", 4)
+                .put("Q2", 1)
+                .put("Q3", 2)
+                .put("Q4", 3);
+        Assert.assertEquals("10", hb.transform("{{sum Q1 Q2 Q3 Q4}}", node));
```

Comment:
Add tests to cover all type conversions, string values and string parsing errors